### PR TITLE
[13.0][MIG] account_move_line_stock_info

### PR DESCRIPTION
**Pull request by ps-tubtim**

### CODE REVIEW

**Comment by rousseldenis** [comment]
@ps-tubtim Could you check travis ?

File: account_move_line_stock_info/tests/test_account_move_line_stock_move.py [hunk 1]
```diff
@@ -0,0 +1,186 @@
+# © 2015 Eficent Business and IT Consulting Services S.L. (www.eficent.com)
```
Comment (AaronHForgeFlow) [inline account_move_line_stock_info/tests/test_account_move_line_stock_move.py]: Many thanks for migrating this. We have a new copyright now. Could you change it please?

`Copyright 2019 ForgeFlow S.L. (https://www.forgeflow.com)`

Thanks in advance

Comment (ps-tubtim) [reply account_move_line_stock_info/tests/test_account_move_line_stock_move.py]: Changed.

**Review by AaronHForgeFlow — COMMENTED** [review]
I miss the product in the account move form view, as in v12:

image

Could you add it?

**Comment by ps-tubtim** [comment]
> I miss the product in the account move form view, as in v12:
> 
> image
> 
> Could you add it?

@AaronHForgeFlow I think that field is many2one field and that move.line from many products, so I can't add it.

File: account_move_line_stock_info/views/stock_move_view.xml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        </field>
+    </record>
+
+
+    <record id="view_picking_form" model="ir.ui.view">
+        <field name="name">stock.picking.form</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//page[3]//group" position="after">
```
Comment (AdriaGForgeFlow) [inline account_move_line_stock_info/views/stock_move_view.xml]: I'm not seeing this in the Picking View (could you also check if a more concrete expression is possible)

**Review by AdriaGForgeFlow — COMMENTED** [review]
Just a small comment below after testing on runbot

File: account_move_line_stock_info/views/stock_move_view.xml (hunk 2 above)
Comment (ps-tubtim) [reply account_move_line_stock_info/views/stock_move_view.xml]: @AdriaGForgeFlow Thanks for review. Edited.

**Review by rousseldenis — APPROVED** [review]
Code review

**Review by AdriaGForgeFlow — APPROVED** [review]
Tested on local :+1:

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: All right

**Comment by CasVissers-360ERP** [comment]
@ps-tubtim 
The migration seems to have messed the picking form view. Can you check?
https://drive.google.com/file/d/1t3gBCZMs_HvvhvC4ATJ2ljuZEhSKKvM9/view

**Pull request merged**